.github/workflows/cleaner.js

```diff
@@ -154,9 +154,19 @@ async function validate(type) {
         console.log(`Repo: ${owner}/${repo}, Stars: ${stars}, Archived: ${isArchived}, Last Commit Date: ${lastCommitDate}`);
         const newOwner = entry.owner.login;
         const newRepo = entry.name;
+
+        let requireUpdate = false;
         if (owner !== newOwner || repo !== newRepo) {
           console.log(`Repo: ${owner}/${repo} has been renamed to ${newOwner}/${newRepo}`);
           content.link = `https://github.com/${newOwner}/${newRepo}`;
+          requireUpdate = true;
+        }
+        // Always update is_archived field
+        if (content.is_archived !== isArchived) {
+          content.is_archived = isArchived;
```

Comment:
I considered only marking archived repositories, but decided to apply it to all entries for better data structure consistency.

Given the potential for future additions such as last update dates, using nested fields might be a better approach like below:

```yml
description: example description
link: https://github.com/hexo/hexo-example
tags:
  - example
repository:
  - is_archived: false
  - last_update: yyyymmdd
  - some_other_field1: ....
  - some_other_field2: ....
```